tools-local/tasks/tasks.proj

```diff
@@ -46,7 +46,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <TasksSrc Include="$(MSBuildThisFileDirectory)**\*.cs*;$(MSBuildThisFileDirectory)**\*.*proj" />
+      <TasksSrc Include="@(ProjectReference)" />
```

Comment:
Hmm, `$(MSBuildThisFileDirectory)**\*.cs*;` is now missing. Give a minute and I will come up with a solution for it.

Reply:
does it matter? that should be handled by the *proj include of the ProjectReference?

Reply:
It matters. MSBuild checks the timestamp of the project files but not of their source code. It doesn't interpret the project files at all for the up-to-date check.